QWeb/internal/actions.py

```diff
@@ -388,7 +388,11 @@ def text_appearance(text: str, **kwargs: Any) -> bool:
 
 @decorators.timeout_decorator_for_actions
 def get_element_text(web_element: WebElement, expected=None, timeout: int = 0) -> Union[bool, str]:  # pylint: disable=unused-argument
-    real_text = web_element.text.strip()
+    real_text = (
+        (web_element.text or "").strip()
+        or (web_element.get_attribute('innerText') or "").strip()
+    )
```

Comment:
Maybe add comment for future readers that `web_element.text` and `web_element.get_attribute('innerText')` can be `None`
Took me a while to understand what is the purpose of empty string ternaries

Reply:
Yep, I did not get it either first but this was changed due to typing catching it.

Reply:
Added comment in a new commit